src/monthlychallenges/oct-2021/index.njk

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Monthly Theme & Challenge for October, 2021: HacktoberFest!'
+description: 'October challenge -> Participate to Hacktoberfest! This month, our members will participate to the DigitalOcean HacktoberFest by being maintainers, contributors, or mentors.'
+layout: layouts/simple.njk
+date: 2021-10-01
+tags: monthlychallenges
+---
+
+
+<h1>
+  <small>Monthly Challenge for October, 2021:</small>
+It's HacktoberFest: participate to Open Source, learn and have fun!</h1>
+
+<p class="lead">
+ This month, we have three tracks. Maintainers will provide issues labeled for HacktoberFest. Contributors will solve issues, and can get support from Mentors. Mentors will help contributors and why not maintainers depending on their area of expertise.
+</p>
+
+<p><strong>Challenge Team Leads & Facilitators</strong>: Aurelie Verrot & Andrew Bush</p>
+
+<hr/>
+
+<h2>Theme</h2>
+<p>The 2021 HacktoberFest!</p>
+
+<h2>Challenge</h2>
+
+<strong>Maintainers</strong>
+<p>.</p>
+
+<strong>Contributors</strong>
+<p>.</p>
+
+<strong>Mentors</strong>
+<p>.</p>
+
+<h2>How to Participate</h2>
+<h4>Before starting</h4>
+<ol>
+  <li>
+    If you are a contributor or a maintainer, make sure to sign up to the <a href="https://hacktoberfest.digitalocean.com/">HacktoberFest official website</a>.
+  </li>
+  <li>
+    Also if you haven't already, sign up to our dashboard: <a href="https://hacktoberfest.virtualcoffee.io/">HacktoberFest Initiative</a>.
+  </li>
+</ol>
+
+<h4>Weekly check-ins</h4>
+<p> Every Friday of the month, come to our Zoom meeting for a check in and share your progress, ask questions, find help and support. To join, find the link in our #monthly-challenge Slack channel.</p>
```

Comment:
```suggestion
<p> Every Friday of the month come to our Zoom meeting to check-in, share your progress, ask questions, and find help and support. To join, find the link in our #monthly-challenge Slack channel.</p>
```
couple text edits.

Reply:
We'll have the Hacktoberfest co-working room this time around, and after thinking it through and talking with Dan and Kirk, it's probably best to talk about Hacktoberfest in the #open-source channel. This will create better use and hopefully prolonged use of the channel--Initially, this was the plan for monthly-challenges (we do a challenge that can get carried over into a channel).

Reply:
It's a good idea actually mostly because it is possible that people continue to contribute after HF, so discussions can continue there.

Reply:
We can post general information abut the challenge in #monthly-challenge, and redirect to check ins Zoom and threads in #open-source.
We can even schedule the posts actually.